micronaut-maven-plugin/src/site/asciidoc/index.adoc

```diff
@@ -9,6 +9,7 @@ This plugin has the following features:
 5. link:examples/aot.html[Using Micronaut AOT].
 6. link:examples/test-resources.html[Integration with Micronaut Test Resources].
 7. link:examples/openapi.html[Integration with Micronaut OpenAPI].
+8. link:generate-import-factory-mojo.html[Importing beans from project dependencies].
```

Comment:
I miss a general explanation like the one you have in the README of your original plugin.

Reply:
Yes, sorry... i was a bit lazy. Will write some examples.